vtex-apps/render-runtime
# feat: improve UX for error pages

#### What does this PR do? \*

Implements new UX for error pages.

#### How to test it? \*

Visit https://anita--storecomponents.myvtex.com/admin/io-manager/explode/?forceLogs=true to see an error inside the iframe, with no request id

and 

https://anita--basedevmkp.myvtex.com/admin for full page error
